oclint-rules/test/naming/LongVariableNameRuleTest.cpp

```diff
@@ -54,3 +54,8 @@ TEST_F(LongVariableNameRuleTest, FiveCharsName)
     testRuleOnCode(new LongVariableNameRule(), "void aMethod() { int iiiii; }",
         0, 1, 18, 1, 22, "Length of variable name `iiiii` is 5, which is longer than the threshold of 3");
 }
+TEST_F(LongVariableNameRuleTest, FifteenCharsName)
+{
+    testRuleOnCode(new LongVariableNameRule(), "void aMethod() { int iiiiiiiiiiiiiii; }",
+        0, 1, 18, 1, 22, "Length of variable name `iiiiiiii[...]` is 15, which is longer than the threshold of 3");
```

Comment:
Could it be truncated in the middle of the string by keeping maybe 3 chars as suffix, like `iiii...iii`? This may not be a good example, but say you have `abcdefghijklmn`, then I am thinking it might be better to have `abcd...lmn`? What do you think?